### DISCUSSION TITLE
When can I join student community and access to student developer pack?

### DISCUSSION THREAD

**Discussion by Hellenqin-uol** [community/community#76639] [GitHub Education]
### Hi everyone,

It is so wordless with Github 2023. 
I got approved for my student status on November 17, 2023, and 6 days passed. Until now I still can not access to student developer pack. It always shows this statement  "Hi, Hellenqin-uol! You were last verified as a student on Nov 17, 2023. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits."
The funniest thing was that I could join the student community and got a student developer pack last year immediately after approval. 
So, my question is, could someone kindly inform me what has happened on Github? Is there a student community and student developer pack on GitHub or is it just a commercial advertisement? 
If yes, I will inform my schoolmates no need to spend time on it. 
Thank you in advance.

**Comment by lotirium** [comment]
The GitHub Student Developer Pack is a legitimate program offered by GitHub, providing free access to various development tools and services for students. It is not merely a commercial advertisement.

If after a reasonable waiting period you still cannot access the pack, the best course of action would be to directly contact GitHub support for assistance. They can provide specific details regarding any changes in policy or issues with access to the Student Developer Pack. Meanwhile, it may be premature to advise others not to apply based on this delay, as the situation might be temporary or specific to your account.

**Comment by Hellenqin-uol** [comment]
Solved. Thanks.
